blog/2024/keycloak-customizable-ui.adoc

```diff
@@ -0,0 +1,43 @@
+:title: Announcing New Keycloak UI Component Libraries!
+:date: 2024-09-04
+:publish: true
+:author: Erik Jan de Wit
+
+We're excited to announce the release of two new npm packages designed to supercharge your Keycloak customization efforts.
+These React component libraries, built on top of PatternFly, provide the essential building blocks for crafting account and admin consoles.
+
+The packages are:
+
+- https://www.npmjs.com/package/@keycloak/keycloak-admin-ui[@keycloak/keycloak-admin-ui]
+  - This package provides the building blocks for creating a Keycloak admin console.
+- https://www.npmjs.com/package/@keycloak/keycloak-account-ui[@keycloak/keycloak-account-ui]
+  - This package provides the building blocks for creating a Keycloak account console.
+- https://www.npmjs.com/package/@keycloak/ui-shared[@keycloak/ui-shared]
+  - This package provides shared components and utilities for building Keycloak UIs.
+
+
+== Accelerate Your Development with Our Quickstart Tool
+
+Kickstart your project with our npm create keycloak-theme my-theme command.
+This streamlined tool generates a project structure, essential dependencies, and configuration, saving you precious time.
+At the moment, the tool is only available for account consoles, but we are working on adding support for admin consoles this will be available in the next release (26.0.0).
+
+== Get Started:
+
+1. Run `npm create keycloak-theme@latest my-theme`.
+1. The keycloak server can be started with `npm run start-keycloak`
+1. Start the development server with `npm run dev`
+1. Customize your theme by editing files in the src directory.
+
+The keycloak server will connect to the development server and all the changes will be reflected in the browser.
+Just open your browser and go to http://localhost:8080/realms/master/account/personalInfo and login with admin/admin.
+This will open the keycloak account console, you will see that the example code has an extra page and some extra content above each page.
```

Comment:
```suggestion
This will open the keycloak account console. You will see that the example code has an extra page and some extra content above each page.
```